test/scripts/run_test.py

```diff
@@ -135,13 +148,11 @@ def run_unit_test(log_dir, device):
     result = True
     for module, unit_test in UNIT_TESTS.items():
         prepare_unit_test(unit_test['path'])
-        install_test(unit_test['path'], device)
-        result_file = run_cases(TestMode.UNIT, module, unit_test['target'], log_dir, device)
-        total, failure = analyse_unit_test_result(result_file)
-        succeed = total - failure
+        result_file = run_cases(TestMode.UNIT, module, unit_test['target'], log_dir, device, unit_test['path'])
```

Comment:
`result_file` is useless for unit test.

Reply:
And please check how to run `gradlew cAT` on a specific module. After changing to using gradle command, the `target` or `path` probably won't be necessary any more.

Reply:
run_cases is a General method for unit test and instrumentation test , target need for instrumentation , and result_file  is recorded logcat for unit test and instrumentation test